packages/components/src/client/utils/mse.ts

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+
+export const autoType = (url: string): string | undefined => {
+  if (url) {
+    switch (true) {
+      case url.endsWith(".m3u8"):
+        return "m3u8";
+      case url.endsWith(".flv"):
+        return "flv";
+      case url.endsWith(".mpd"):
+        return "mpd";
+      default:
+        return "mp4";
+    }
+  }
+
+  return undefined;
+};
+
+export const Hls = async () => {
+  const { default: Hls } = await import("hls.js/dist/hls.min.js");
+
+  return Hls;
+};
+
+export const MpegTs = async () => {
+  const { default: mpegts } = await import("mpegts.js/dist/mpegts.js");
+
+  return mpegts;
+};
```

Comment:
```suggestion
export const MpegTs = import("mpegts.js/dist/mpegts.js").default
```

So why is this function needed? Also I think you should use the magic comment `/* webpackChunk: xxx */ to ensure releated modules are packed to one chunk.

Reply:
I think there is something wrong here, I should not export, I will rewrite this part of the code